util.sh

```diff
@@ -8,21 +8,68 @@ RED='\033[0;31m'
 BLUE='\e[104m'
 NC='\033[0m' # No Color
 
+pod_query_ready() {
```

Comment:
I'm also wondering if it's not worth including the wait loop in the implementation of these functions, since all the calls currently do a: `until xxx_ready [...]; do sleep 1; done`. What do you think?

Reply:
> shouldn't it be pod_ready? 

There's a subtile difference in that the input is a label selector rather than a resource name.

> I'm also wondering if it's not worth including the wait loop in the implementation of these functions

I considered doing that, if we renamed to wait_for_*_ready we could include the `util; sleep done`. I left it out because maybe we'll need a different kind of sleep in the future, but we haven't so far.

Reply:
👍